From 38e128d799217a08ea12ddf75a6a7ed440384b5f Mon Sep 17 00:00:00 2001
From: Ander Conselvan de Oliveira <ander.conselvan.de.oliveira@intel.com>
Date: Thu, 18 Jul 2013 15:11:24 +0300
Subject: [PATCH 2/4] gallium-egl: Don't add a listener for wl_drm twice in
 wayland platform

A listener is added just after the interface is bound, in
registry_handle_global().

Signed-off-by: Ander Conselvan de Oliveira <ander.conselvan.de.oliveira@intel.com>
---
 src/gallium/state_trackers/egl/wayland/native_drm.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gallium/state_trackers/egl/wayland/native_drm.c b/src/gallium/state_trackers/egl/wayland/native_drm.c
index a4bcdd8..e1aeeeb 100644
--- a/src/gallium/state_trackers/egl/wayland/native_drm.c
+++ b/src/gallium/state_trackers/egl/wayland/native_drm.c
@@ -227,7 +227,6 @@ wayland_drm_display_init_screen(struct native_display *ndpy)
    if (wayland_roundtrip(&drmdpy->base) < 0 || drmdpy->wl_drm == NULL)
       return FALSE;
 
-   wl_drm_add_listener(drmdpy->wl_drm, &drm_listener, drmdpy);
    if (wayland_roundtrip(&drmdpy->base) < 0 || drmdpy->fd == -1)
       return FALSE;
 
-- 
1.8.4

